test/benchmark/lmbench/fifo-latency/config.json

```diff
@@ -3,5 +3,7 @@
     "alert_tool": "customSmallerIsBetter",
     "search_pattern": "Fifo latency",
     "result_index": "3",
-    "description": "The latency of fifo on a single processor."
-}
+    "description": "lat_fifo",
+    "title": "[FIFO] The cost of write+read (1B)",
+    "display": "true"
```

Comment:
Does display have a default value? I think most benchmark should have `display` equal to `true`, right? So I think you only need to add this field when the benchmark should NOT be displayed.

Reply:
Yes. The workflow will treat it as true if it is not set.

Reply:
I will avoid explicitly declaring it in `config.json` and mention it in the README.